packages/core/src/components/hero/hero.css

```diff
@@ -0,0 +1,252 @@
+.hds-hero {
+  --background-color: var(--color-white);
+  --color: var(--color-black-90);
+  --image-position: bottom right;
+  --koros-color: var(--color-white);
+  --angled-koros-inset: auto auto 10% -33%;
+  --horizontal-padding-small: var(--spacing-layout-2-xs);
+  --horizontal-padding-medium: var(--spacing-layout-xs);
+  --horizontal-padding-large: var(--spacing-layout-xs);
+  --horizontal-padding-x-large: var(--spacing-layout-xs);
+  --padding-horizontal: var(--horizontal-padding-small);
+  --padding-vertical: var(--spacing-xl);
+  color: var(--color);
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  z-index: 1;
+  background-color: var(--background-color);
+}
+.hds-hero h1 {
+  font-size: var(--fontsize-heading-xl);
+  letter-spacing: -1px;
+  font-weight: normal;
+  line-height: 1em;
+  margin-top: 0;
+}
```

Comment:
Maybe just Hero.Title and Hero.Text. But forcing devs to use those seems useless. What is the gain?

Reply:
Main gain is just small optimisation because of browser selector matching from right to left (first match all h1/p elements, then the parent class).

Here's some other thoughts:
1. Hero.Title and Hero.Text give users clear elements on what to use and what works here
2. Guiding users to use h1 and p tags here sort of gives the impression that you can put any HTML here or use h2 or spans instead. The usable elements could be mentioned in docs but I don't know if that's as powerful as own specific components for them.
3. No other core component uses heading or p tags, maybe because of the browser selector matching thing. Can @minevala confirm or speculate on this? 😄

Reply:
`Hero.Title` would also hide the element type and devs should always check docs/source what's the element so the headings are in the correct order.

Reply:
Of course `Hero.H1` can be done, but still seems silly.